src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -182,6 +182,21 @@
         <target state="translated">, </target>
         <note>Separator used for separating list of platform names: {API} is only supported on: {‘windows’, ‘browser’, ‘linux’}</note>
       </trans-unit>
+      <trans-unit id="CuriouslyRecurringTemplatePatternDesciption">
+        <source>Curiously Recurring Template Interface requires the derived type used for 'TSelf' type parameter</source>
```

Comment:
Similar comment here. I wouldn't use the general terms `Curiously Recurring Template Interface`, but instead something more targeted to the scenario we are verifying

```suggestion
        <source>'{0}' Interface requires the derived type used for 'TSelf' type parameter</source>
```

Where `{0}` is the interface being implemented.

Reply:
We cannot use a placeholder in a description, so updated it to `Specific generic math interfaces requires the derived type used for the 'TSelf' type parameter` (the message has placeholder for the interface name)